js_modules/dagster-ui/packages/ui-core/src/launchpad/useLaunchMultipleRunsWithTelemetry.ts

```diff
@@ -49,7 +50,16 @@ export function useLaunchMultipleRunsWithTelemetry() {
           opSelection: undefined,
         };
 
-        const result = (await launchMultipleRuns({variables})).data?.launchMultipleRuns;
+        const finalized = {
+          ...variables,
+          executionParamsList:
+            variables.executionParamsList instanceof Array
```

Comment:
Prefer `Array.isArray(...)` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#description

Reply:
Oh nice did not know about this actually!